### ISSUE TITLE
Add Mass() expression to the Physics Behavior

### ISSUE THREAD

**Issue by tristanbob** [4ian/GDevelop#2714]
## Description

Users might want to add a Physics force that adapts based on the mass of the object.  The mass of the object can be calculated by:  CollisionMaskArea * Physics2::Density()

There is currently no way to get the area of the physics collision mask, so the Mass cannot be calculated.

## Solution suggested

Create new expressions:

- [ ] Object.Physics2::Mass()
- [ ] Object.Physics2::CollisionMaskArea() 

(from the Box2D manual)
https://box2d.org/documentation/md__d_1__git_hub_box2d_docs_dynamics.html

> The body's mass data is available through the following functions:
> 
> float b2Body::GetMass() const;
> float b2Body::GetInertia() const;
> const b2Vec2& b2Body::GetLocalCenter() const;
> void b2Body::GetMassData(b2MassData* data) const;

**Comment by Bouh** [comment]
` Object.Physics2::CollisionMaskArea()` this area can be a rectangle AND a **custom shape**.

**Comment by tristanbob** [comment]
> ` Object.Physics2::CollisionMaskArea()` this area can be a rectangle by default AND a **custom shape**, this have to be taken into account.

Yes, exactly.  The default physics collision mask is the AABB rectangle, but it can be changed by the user to be a different rectangle or a circle (and perhaps other shapes, in the future).
